google/cloud/spanner_v1/database.py

```diff
@@ -726,6 +727,15 @@ def execute_pdml():
 
                 return result_set.stats.row_count_lower_bound
```

Comment:
I am not able to add comment in above line, hence adding comment here
pass `trace_name="CloudSpannerOperation.ExecuteStreamingSql"` at https://github.com/googleapis/python-spanner/blob/054a18658eedc5d4dbecb7508baa3f3d67f5b815/google/cloud/spanner_v1/database.py#L717

Reply:
Done and thank you @harshachinta